inst/report/comparison_report.Rmd

```diff
@@ -165,13 +165,13 @@ if (print_en) {
 }
 
 if (print_fr) {
-  cat(paste0("## ", country, " Subnational HIV Estimates \n"))
-  cat(paste0("Naomi estimates for **", quarter_t2, "** (report generated ", report_date, ")"))
+  cat(paste0("## ", country, " Estimativas subnacionais do VIH \n"))
+  cat(paste0("Les estimations de Naomi pour **", quarter_t2, "** (rapport généré ", report_date, ")"))
 }
 
 if (print_pt) {
-  cat(paste0("## ", country, " Subnational HIV Estimates \n"))
-  cat(paste0("Naomi estimates for **", quarter_t2, "** (report generated ", report_date, ")"))
```

Comment:
We didn't get a string for these from UNAIDS, we should send this to them to check

Reply:
These come from the summary report - have updated.